telegrambots/pom.xml

```diff
@@ -80,8 +80,25 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
-
+      <repositories>
+    <repository>
+        <id>spigot-repo</id>
+        <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
```

Comment:
Spigot repository. Can be replaced with bukkit, but imho spigot will live longer